src/posts/2020-09-19-how-to-write-accessible-forms.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: Create accessible forms
+description: Practial tips on how to create accessible forms.
+category: How-to
+author: Hamsa Harcourt
+date: 2020-09-16
+further_reading:
+  - title: Error Messages
+    url: https://www.w3.org/WAI/EO/Drafts/tutorials/forms/error-messages/
+    source:  Web accessibility tutorials 
+    year: 2014
+  - title: Provide helpful error messages
+    url: https://accessibility.huit.harvard.edu/provide-helpful-error-messages
+    source:  Havard university
+    year: 2014
+  - title: HTML label element reference
+    url: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label
+    source: MDN web docs
+    year: 2020
+  - title: Don’t Use The Placeholder Attribute
+    url: https://www.smashingmagazine.com/2018/06/placeholder-attribute/
+    source: Smashing Magazine
+    year: 2018
+  - title: "HTML5 Accessibility Chops: the placeholder attribute"
+    url: https://developer.paciellogroup.com/blog/2011/02/html5-accessibility-chops-the-placeholder-attribute/
+    source: The Paciello Group
+    year: 2011
+  - title: Using @placeholder on Input
+    url: https://www.w3.org/WAI/GL/wiki/Using_@placeholder_on_input
+    source: W3C
+    year: 2012
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - howto
+---
+
+With the world becoming data reliant, it's no surprise how data is the new gold. With the right data, you can improve peoples' lives, find solutions to problems and so on. There are many ways to capture peoples' data but on the web, forms are your go-to solution.
+
+Here are some  practical tips on how to write accessible forms.
+
+
+## Always label your forms
+
+Labels are a great way to get started with writing accessible forms. They are vital for helping people who use [screen readers](https://en.wikipedia.org/wiki/Screen_reader) and other kinds of assistive technology determine what label text go with what input field.
+
+To associate a `<label>` with an `<input>`, set the identifier(ID) in the `<input>` to correspond with the `for` attribute of the `<label>`. This is the generally accepted approach.
+```html
+<form>
+  <label for="firstName">First Name</label>
+  <input type="text" id="firstName" name="firtName"/>
+</form>
+```
+
+
+**Note**: It is adviceful not to nest `<input>` directly inside `<label>` because some assistive devices (such as [Dragon NaturallySpeaking](https://en.wikipedia.org/wiki/Dragon_NaturallySpeaking)) do not support it.

```

Comment:
```suggestion
**Note**: Do not to nest `<input>` inside of `<label>` because some assistive technology (such as [Dragon NaturallySpeaking](https://en.wikipedia.org/wiki/Dragon_NaturallySpeaking)) do not support it.
```